src/Git/Serialise.hs

```diff
@@ -150,7 +150,7 @@ instance GitObject Commit where
         let
           tm = floor $ utcTimeToPOSIXSeconds $ zonedTimeToUTC t :: Int
           tzm = timeZoneMinutes $ zonedTimeZone t
-          (h, m) = divMod tzm 60
+          (h, m) = divMod (abs tzm) 60
```

Comment:
Can you get printf to always write sign?

Reply:
Hmm, turns out it's not so simple because `(-100) ``divMod`` 60` is `(-2,20)` rather than `(-1, -40)`. In any case, neither of those is quite what we want anyway. Might as well stick with what we've got, because it works!

Reply:
sounds sensible to me.